code/script.js

```diff
@@ -1,44 +1,154 @@
 
+let questionNumber = 1;
+
+let userName = ""; //this means that i want userNamne to be available in any of our functions (and not only in the funciotn where it is declared)
 
 // Declare your functions after this comment
 
 // This function will add a chat bubble in the correct place based on who the sender is
 const showMessage = (message, sender) => {
   // the if statement checks if the sender is 'user' and if that's the case it inserts an html senction inside the chat with the posted message
-  if (sender === 'user') {
+  if (sender === "user") {
     chat.innerHTML += `
       <section class="user-msg">
         <div class="bubble user-bubble">
           <p>${message}</p>
         </div>
         <img src="assets/user.png" alt="User" />  
       </section>
-    `
+    `;
     // the else if statement checks if the sender is a bot and if that's the case it inserts an html senction inside the chat with the posted message
-  } else if (sender === 'bot') {
+  } else if (sender === "bot") {
     chat.innerHTML += `
       <section class="bot-msg">
         <img src="assets/bot.png" alt="Bot" />
         <div class="bubble bot-bubble">
           <p>${message}</p>
         </div>
       </section>
-    `
+    `;
   }
   // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
   chat.scrollTop = chat.scrollHeight;
-}
+};
 
+//const nextQuestion = (message) => {};
 // Starts here
 const greetUser = () => {
   // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot');
+  showMessage("Hello there, What's your name?", "bot");
   // Just to check it out, change 'bot' to 'user' here 👆
-}
+};
+
+// Store the value in a variable so I can access it after we
+
+// Initial button click, here I should get the name entered
+sendButton.addEventListener("click", (event) => {
+  event.preventDefault();
+  userName = name.value; //username is here reassigned to mean what we want it to mean in this specific function
+  showMessage(`${userName}`, "user");
+  name.value = ""; // clear it from the input
+  setTimeout(() => askZodiacSigns(), 1000); //calls the function askZodiac
+});
+
+//Here I call the function where I welcome the user and ask them to tell me their zodiac sign
+const askZodiacSigns = () => {
+  console.log("this works");
+  showMessage(
+    `OMG ${userName} is my favourite name in the whole world 🥳!! What's your zodiac sign?`,
+    "bot"
+  );
+  setTimeout(() => nextQuestion(), 1000); //calls the function nextQuestion
+};
+
+const nextQuestion = () => {
+  //this function is whats defining what we meen with the function nextQuestion
+  console.log("next question works!");
+  /*  showMessage(´next question is working´ , "user"); */
+
+  inputWrapper.innerHTML = `
```

Comment:
Here you have indicated an inputWrapper.innerHTML which suggests to me you want to add buttons, is this correct? In which case I think you could more simply write 

```
inputWrapper.innerHTML = `
<button id=“zodiacBtn">Zodiac </button>
`
```